components/eventing-controller/cmd/eventing-controller-nats/main.go

```diff
@@ -79,6 +83,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
```

Comment:
I would change the hardcoded string literal to a constant

Reply:
Done